containers/flatpak/cockpit-session

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo '78
+
+{"command":"authorize","cookie":"session6130231636660820", "challenge": "*"}'
```

Comment:
What does this even do? If the flatpak even cares about a "local" (to the container) session, it should just call cockpit-ws with `--local-session=/usr/bin/cockpit-bridge`. But I thought the idea was to use cockpit-ssh as the session helper anyway?

Reply:
So basically, `login.js` contains "autologin" code to try and login on first page load (thinking maybe krb will help us).  That already hits cockpit-session.  If cockpit-session isn't available, then you'll get "Internal error in login process" and you're stuck.

This ugly little trick gets us far enough to see an actual login page.

I think the real solution here is probably to avoid autologin in the case that the `RequireHost` parameter is `true` in the config.  Then we can drop this ugly hack.

Reply:
I still don't understand -- with `--local-session` it should not even *try* to call cockpit-session. Likewise, cockpit.conf could just disable basic auth. We don't expect to ever actually do authentication, right? As the flatpak runs as user, and we want to talk to cockpit-bridge of that very same user.